website/docs/docs/cloud/git/setup-azure-service-principal.md

```diff
@@ -0,0 +1,173 @@
+
+
+## Migrate to service principal
+
+Migrate from a service user to service principal using the existing app. It will only take a few steps and you won't experience any service disruptions. 
+
+- Verify whether or not your app has a service principal
+    - If not, create the app service principal
+- Update the application's configuration
+- Update the configuration in dbt Cloud
+
+### Verify the service principal
+
+You will need an Entra ID admin to complete these steps.
+
+To confirm whether your existing app already has a service principal:
+ 
+1. In the Azure account, navigate to **Microsoft Entra ID** -> **Manage** -> **App registrations**.
+2. Click on the application used for the service user integration with dbt Cloud. 
+3. Verify whether there is a name populating the **Managed application in local directory** field. 
+    - If there is a name, continue on to the next step from the [add role to your service principal](#add-a-role-to-service-principal) and follow the remaining instructions.
+    - If there is no name, go to the next section, [Create the service principal](#create-the-service-principal).
+4. Follow the instructions to [Add permissions](#add-permissions-to-your-service-principal) to your service principal.
+5. In your dbt Cloud account:
+    1. Navigate to **Account settings** and click **Integrations**
+    2. Click the **edit icon** to the right of the **Azure DevOps** settings.
+    3. Change **Service user** to **Service principal** and click **Save**. You do not need to edit any existing fields.
+
+### Create the service principal
+
+If there is not a name populating that field, this means a Service Principal does not exist. To configure a Service Principal, please review the instructions here.
+
+If your dbt Cloud app does not have a service principal, take the following actions in your Azure account:
+
+1. Navigate to **Microsoft Entra ID**.
+2. Under **Manage** on the left-side menu, click **App registrations**.
+3. Click the app for the dbt Cloud and Azure DevOps integration.
+4. Locate the **Managed application in local directory** field and click **Create Service Principal**. 
+
+    <Lightbox src="/img/docs/cloud-integrations/create-service-principal.png" width="80%" title="Example of the 'Create Service Principal' option highlighted ."/>
+
+4. Follow the instructions to [Add permissions](#add-permissions-to-your-service-principal) to your service principal.
+5. In your dbt Cloud account:
+    1. Navigate to **Account settings** and click **Integrations**
+    2. Click the **edit icon** to the right of the **Azure DevOps** settings.
+    3. Change **Service user** to **Service principal** and click **Save**. You do not need to edit any existing fields.
+
+Continue on from the [add role to your service principal](#add-a-role-to-service-principal) section on this page and follow the remaining instructions.
```

Comment:
Can we update this line and include the other section that is relevant here as well which is to connect azure devops. Following the remaining instructions is misleading since it will also include adding the app details to dbt cloud which we don't want the migration flow to include